auth/tenant_mgt.go

```diff
@@ -198,6 +200,33 @@ func (tm *TenantManager) DeleteTenant(ctx context.Context, tenantID string) erro
 	return err
 }
 
+// Tenants returns an iterator over tenants in the project.
+//
+// If nextPageToken is empty, the iterator will start at the beginning. Otherwise,
+// iterator starts after the token.
+func (tm *TenantManager) Tenants(ctx context.Context, nextPageToken string) *TenantIterator {
+	it := &TenantIterator{
+		ctx: ctx,
+		tm:  tm,
+	}
+	it.pageInfo, it.nextFunc = iterator.NewPageInfo(
+		it.fetch,
+		func() int { return len(it.tenants) },
+		func() interface{} { b := it.tenants; it.tenants = nil; return b })
```

Comment:
Nit: Should the closing `)` be in a new line?

Reply:
We've used this style in other files (provider_config.go, user_mgt.go). So my preference would be to keep it this way.